src/main/java/hu/dpc/rt/kafkastreamer/exporter/KafkaExporterClient.java

```diff
@@ -74,9 +76,11 @@ public void index(final Record<?> record) {
 
         if (configuration.shouldIndexRecord(record)) {
             logger.trace("sending record to kafka: {}", record.toJson());
+            JSONObject jsonObject = new JSONObject(record.toJson());
```

Comment:
[]check if we can do this more efficiently in terms Time-Complexity.

Reply:
Updated the code to achieve the same through String Manipulation instead of JsonObject.